modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C

```diff
@@ -99,7 +99,8 @@ ComputeLinearElasticPFFractureStress::computeStrainSpectral(Real & F_pos, Real &
     nval += eneg[i] * eneg[i];
   }
 
-  _stress[_qp] = stress0pos * _D[_qp] + stress0neg;
+  _stress[_qp] = stress0pos * _D[_qp] -
+                 _pressure[_qp] * RankTwoTensor(RankTwoTensor::initIdentity) * _I[_qp] + stress0neg;
```

Comment:
The identity tensor is used multiple times. Maybe declare a rank two tensor earlier and initialize it to identity? i.e.
`RankTwoTensor I2(RankTwoTensor::initIdentity);`